src/rgw/rgw_sal_motr.cc

```diff
@@ -3920,4 +3920,4 @@ void *newMotrStore(CephContext *cct)
   return store;
 }
 
-}
+}
```

Comment:
Revert this change, please.

Reply:
You have removed the empty line at the end. Please revert this.

Reply:
Reverted